src/Microsoft.TemplateEngine.Edge/TemplateListFilter.cs

```diff
@@ -14,7 +16,19 @@ namespace Microsoft.TemplateEngine.Edge
 {
     public static class TemplateListFilter
     {
-        // Deprecating. ITemplateMatchInfo will eventually fully replace IFilteredTemplateInfo
+        /// <summary>
+        /// Exact match criteria - the templates should match all filters.
+        /// </summary>
+        /// <seealso cref="GetTemplateMatchInfo"/>
+        public static Func<ITemplateMatchInfo, bool> ExactMatchFilter => x => x.IsMatch;
+
+        /// <summary>
+        /// Partial match criteria - the templates should match one of the filters.
```

Comment:
`should match one of the filters` could be `should match at least one of the filters`